### ISSUE TITLE
Foxy ability miscalculation

### ISSUE THREAD

**Issue by Tricky1975** [Tricky1975/Star-Story#509]
It appears the number of moves for Foxy is not entirely correctly set up.
This causes suffering in:
- [x] The "???" in Foxy's ability list to disappear too early.
- [x] Does the "I'm sexy and I know it" achievement work the way it should?
- [x] Foxy will never teach "Pickpocket" to Wendicka
- [x] I'm also not sure if the "Dragon Charge -> Sky Dragon Slash" combo can ever be obtained this way.

This require full investigation

**Comment by Tricky1975** [comment]
Considering how the code is written teaching Foxy her new moves, I'm not worried about the last one.

**Comment by Tricky1975** [comment]
Riiiiight.... I calculated 7 abilities for her, but forgot about the two she already has at the start.
Foxy could teach "Pickpocket" to Wendicka, but only if she unlocked 6 abilities exactly, which was the wrong number (it had to be 8), but any bugs would also rule things out this way. I came up with better code for that.

Anyway everything should be fixed now, and I'll push the fixes through right now. ;)
